pom.xml

```diff
@@ -18,6 +18,9 @@
     <jenkins.version>2.235.5</jenkins.version>
     <java.level>8</java.level>
     <configuration-as-code.version>1.51</configuration-as-code.version>
+    <aws-sdk.version>1.11.687</aws-sdk.version>
+    <aws-credentials.version>1.26</aws-credentials.version>
```

Comment:
was curious about both, more so the sdk since it's a bigger change (figured it added all that stuff to the EmptyEC2 file). Don't worry about reverting the credentials

Reply:
According to `git blame` on https://github.com/aws/aws-sdk-java/blame/master/aws-java-sdk-autoscaling/src/main/java/com/amazonaws/services/autoscaling/model/LaunchTemplateOverrides.java . The `weightedCapacity` was added in that version